docs/src/configuration.md

```diff
@@ -34,46 +35,34 @@ For example:
-```json
-{
-  "name": "String"
-}
-```
-
-### Description
-
-The description set will be used in the description tag of your website and this is mostly used for SEO.
-
-By default Oranda will always look for this value in your `package.json`/`cargo.toml` and use it unless the value in the `oranda.json` overrides it.
-
-```json
-{
-  "description": "String"
-}
-```
-
-### Repository
-
-### Homepage
-
-The homepage set will be used in the meta tags of your website.
-
-By default Oranda will always look for this value in your `package.json`/`cargo.toml` and use it unless the value in the `oranda.json` overrides it.
-
-```json
-{
-  "homepage": "String"
-}
-```
+> **NOTE:** All paths in `oranda.json` are relative to the `oranda.json` file. We
+  recommend placing this file in the root of your project, alongside any other
+  project manifest files you may be using, such as a `package.json` or `Cargo.toml`.
+
+## Configuration options
+
+Some options have more detailed documentation, indicated by a link from the configuration key.
+
+- `name`: Your project's name. By default fetched from your project's manifest file.
+- `description`: Your project's description. By default fetched from your project's manifest file.
+- `dist_dir`: The directory where your static files will be output to. By default, this is `public/`.
+- `homepage`: Your project's homepage. By default fetched from your project's manifest file.
+- `static_dir`: Static content that oranda will copy to its output folder. By default, this is `static/`.
+- `no_header`: Whether to hide the page header (title and navigation bar).
+- `readme_path`: The path to your project's readme file. oranda by default tries a variation of the standard `README.md`.
+- `repository`: Your project's Git repository. By default fetched from your project's manifest file.
+- [`analytics`](./configuration/analytics.md): Configuration for page analytics.
+- [`additional_pages`](./configuration/additional-pages.md): An object of additional Markdown pages that you'd like to be included. All of these will appear in the site
+  header.
+- [`social`](./configuration/social.md): Options useful for SEO features.
+- [`artifacts`](./configuration/artifacts.md): Configuration for enabling downloadable artifacts, as well as the `cargo-dist`
+  integration.
+- `version`: Not currently used.
+- `logo`: Path to a custom logo to be shown in your website header.
+- `favicon`: Path to a custom favicon.
+- `path_prefix`: If you're hosting oranda on a nested path (e.g. `mysite.cool/myproject`), you should set `path_prefix` to
+  `myproject` in your configuration in order for oranda to generate correct links.
+- `license`: Your project's license. By default fetched from your project's manifest file.
+- [`mdbook` or `md_book`](./configuration/mdbook.md): Configuration for mdbook.
+- [`changelog`](./configuration/changelog.md): Enable changelog generation.
+- [`styles`](./configuration/theme.md): Configuration for custom styles or themes.
```

Comment:
Some things that might be worth stealing [from cargo-dist's config docs](https://opensource.axo.dev/cargo-dist/book/config.html):

* consistent example value
* version the config key was introduced in